app/Collections/Product.php

```diff
@@ -390,7 +390,7 @@ public function generate_typesense_data($product)
 				// Add variations data
 				'crossSellData' => empty($cross_sell_data) ? $related_products : $cross_sell_data,
 				'upsellData' => $upsell_data,
-				'additionalTabs' => apply_filters('wooless_product_tabs', $formatted_additional_tabs, $product_id),
+				'additionalTabs' => apply_filters('wooless_product_tabs', $formatted_additional_tabs, $product_id, $product),
```

Comment:
that might be needed in the future  jax because inside get_custom_tabs we might need to get other product information instead of just product id. I do think $product_id is not needed on this filter because people who need to get  other product information need to refetch the product information. This way we don't need to refetch product information in the futture.